Tests/Validator_test/ValidatorTest.cs

```diff
@@ -12,19 +12,51 @@ public ValidatorTest()
     {
         _validator = new Validator();
     }
+    
+    // Var lidt usikker på om det her er fint, kunne ikke få det til at virke med inline data
+    public static IEnumerable<object[]> GetDescriptionTestData()
+    {
+        yield return new object[] { "Test", false }; // 4 characters
+        yield return new object[] { "Tests", true }; // 5 characters
+        yield return new object[] { new string('a', 255), true }; // 255 characters
+        yield return new object[] { new string('a', 256), false }; // 256 characters
+    }
+
+    [Theory]
+    [MemberData(nameof(GetDescriptionTestData))]
```

Comment:
Som jeg forstår dette går det i mod best practice af test da der er noget logik i objekterne som gøre at du kan både teste for valid og invalid scenarier inputs samtidig. så jeg ville lave det mere simplet.